### PR TITLE
feat(sendgrid): Add installPlans for SendGrid Event Webhook and S3 log ingestion

### PR DESCRIPTION
**Pull request by ktst79**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

**Comment by ktst79** [comment]
Can I know the status of this PR?

File: install/aws/aws-log-ingestion-s3/install.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+id: aws-log-ingestion-s3
+name: AWS Lambda for sending logs from S3
+title: AWS Lambda for sending logs from S3
+description: |
+  You can send your Amazon S3 buckets to New Relic using our AWS Lambda function, NewRelic-log-ingestion-s3
```
Comment (jpvajda) [inline install/aws/aws-log-ingestion-s3/install.yml:5]:
```suggestion
  You can send your Amazon S3 buckets to New Relic using our AWS Lambda function, NewRelic-log-ingestion-s3.
```

File: install/aws/aws-log-ingestion-s3/install.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: link
+  destination:
+    url: >-
+      https://docs.newrelic.com/docs/logs/forward-logs/aws-lambda-sending-logs-s3/
```
Comment (jpvajda) [inline install/aws/aws-log-ingestion-s3/install.yml:14]:
```suggestion
      https://docs.newrelic.com/docs/logs/forward-logs/aws-lambda-sending-logs-s3/
      
```

File: install/third-party/sendgrid/install.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+id: third-party-sendgrid-integration
+name: sendgrid
+title: SendGrid Events Webhook
+description: |
+    SendGrid provides a cloud-based service that assists businesses with email delivery. In order to integrate SendGrid Events with New Relic, you will need to send SendGrid events by using the SendGrid Event Webhook fist. If you are already sending events to AWS S3 using the SendGrid Event Webhook, they can be transferred to New Relic by using NewRelic-log-ingestion-s3.
```
Comment (jpvajda) [inline install/third-party/sendgrid/install.yml:5]:
```suggestion
    SendGrid provides a cloud-based service that assists businesses with email delivery. In order to integrate SendGrid Events with New Relic, you will need to send SendGrid events by using the SendGrid Event Webhook. If you are already sending events to AWS S3 using the SendGrid Event Webhook, these events can be transferred to New Relic by using NewRelic-log-ingestion-s3.
```

File: install/third-party/sendgrid/install.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: link
+  destination:
+    url: https://docs.sendgrid.com/for-developers/tracking-events/getting-started-event-webhook
```
Comment (jpvajda) [inline install/third-party/sendgrid/install.yml:14]:
```suggestion
    url: https://docs.sendgrid.com/for-developers/tracking-events/getting-started-event-webhook
    
```

**Review by jpvajda — CHANGES_REQUESTED** [review]
Thanks for the contribution, just a few suggestions here.

**Review by jpvajda — APPROVED** [review]

**Comment by ktst79** [comment]
Thanks! @jpvajda

**Comment by jpvajda** [comment]
@ktst79 Thank you for the contribution! 🚀

**Pull request merged**